third_party/egeria/deploy.sh

```diff
@@ -4,24 +4,22 @@ set -e
 set -x
 
 : ${WITHOUT_OPENSHIFT=false}
+: ${DONOT_DELETE_PERSISTENT_STORAGE=false}
 
-# CHART=odpi-egeria-lab
-# RELEASE=lab
-NAMESPACE=egeria-catalog2
+NAMESPACE=irltest1
```

Comment:
what are we using "irltest1" for? if this is the target namespace where we install Egeria it should not be named IRLTEST1

Reply:
Sorry this was an overlooking error. Corrected it. Please check. Thanks.